.github/workflows/tests.yml

```diff
@@ -104,6 +104,44 @@ jobs:
           env_vars: PYTHON
           name: ${{ matrix.python }}
 
+  msvc:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - '3.11'
+        meson:
+          -
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up target Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Install Ninja
+        run: python -m pip --disable-pip-version-check install ninja
+
+      - name: Setup MSVC
+        uses: bus1/cabuild/action/msdevshell@v1
```

Comment:
Since this seems to be from a GitHub user that we don't necessarily want to trust, I suggest changing this to:
```
uses: bus1/cabuild/action/msdevshell@e22aba57d6e74891d059d66501b6b5aed8123c4d  # v1
```
following https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
The user in question is one of the maintainers of dbus-broker, among other things, and a known figure in the open source world. I trust him much more than, say, codecov. Howrver, I'll explicitly pin the action to the git revision.

Reply:
Thanks. Yep, I'm also not a fan of codecov. I think trusting https://github.com/actions/ makes sense (and is more convenient than pinning), maybe a few other well-known orgs, and the rest needs pinning.

Reply:
Makes sense. It is pinned now.